src/style.css

```diff
@@ -0,0 +1,98 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.text1 {
+  width: 100%;
+  height: 70px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding-left: 62px;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.label1 {
+  position: relative;
+  display: block;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.image1 {
+  position: absolute;
+  display: block;
+  left: 26px;
+  top: 25px;
+  width: 19px;
+  height: 19px;
+}
+
+.text1:focus {
+  outline: none;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 900;
+}
+
+.text1:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.text2 {
+  font-family: "Avenir", sans-serif;
+  width: 100%;
+  line-height: 40px;
+  padding: 0 33px;
+  font-size: 14px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-weight: 300 ;
+}
```

Comment:
looks like a lot of rules are repetitive here. Take a look at the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) p.2 again

Reply:
please can you tell me more about it ? and how to fix it ?

Reply:
done :)